quil-py/quil/program/__init__.pyi

```diff
@@ -175,6 +175,121 @@ class Program:
         while ensuring that unique value is not already in use within the program.
         """
         ...
+    def control_flow_graph(self) -> "ControlFlowGraph":
+        """
+        Return the `control flow graph`_ of the program.
+
+        .. _control flow graph: https://en.wikipedia.org/wiki/Control-flow_graph
+        """
+
+@final
+class BasicBlock:
+    def as_fixed_schedule(self, program: Program, include_zero_duration_instructions: bool) -> FixedSchedule:
```

Comment:
Does `include_zero_duration_instructions` have a reasonable default? If so, might be more pythonic to make it an optional kwarg.

Reply:
good point; at @mhodson-rigetti 's recommendation though I'll just end up removing it